variables.tf

```diff
@@ -477,4 +477,18 @@ variable "gitlab_hpa_max_replicas_webservice" {
   description = "Set the maximum hpa pod replicas for the Gitlab webservice."
   default     = 10
 }
- 
\ No newline at end of file
+
+######################
+# MONITORING SECTION #
+######################
+
+variable "uptime_monitoring_path" {
+  type        = string
+  description = "The path to the page to run the check against."
+  default     = "/users/sign_in"
```

Comment:
We should check here if we can reuse those endpoints instead of going to sign_in by default: https://docs.gitlab.com/ee/user/admin_area/monitoring/health_check.html

Reply:
changed default to: "/-/liveness"